app/decorators/course_decorator.rb

```diff
@@ -130,6 +130,27 @@ def sorted_subjects
     object.subjects.map(&:subject_name).sort.join("<br>").html_safe
   end
 
+  def chosen_subjects(course)
+    if course.master_subject_id.nil?
+      sorted_subjects
+    else
+      main_subject_id = course.master_subject_id
+      main_subject = Subject.find(main_subject_id)
+      raw_additional_subjects = course.subjects.map { |subject| subject unless subject.id == main_subject_id }
+      additional_subjects = raw_additional_subjects.compact
+
+      # The below is to ensure modern language subjects chosen appear above the second main subject chosen
+
+      if main_subject_id == 33
```

Comment:
Could we fetch the id instead of hardcoding

Reply:
Not sure what you mean, something like this? 🤷🏼‍♂️ 

https://github.com/DFE-Digital/publish-teacher-training/pull/3113/commits/d5c39d3e57c18670f28f955f19bf047892573367

Reply:
I think you can do `SecondarySubject.physics.id` which is available as a method

Reply:
lol - oh right. Makes sense. Done